src/Boundaries/gasexchange.jl

```diff
@@ -45,17 +40,48 @@ adapt_structure(to, pCO₂_model::pCO₂) = pCO₂(adapt(to, pCO₂_model.solubi
                                              adapt(to, pCO₂_model.water_dissociaiton),
                                              adapt(to, pCO₂_model.lower_pH_bound),
                                              adapt(to, pCO₂_model.upper_pH_bound),
-                                             adapt(to, pCO₂_model.boron_ratio),
-                                             adapt(to, pCO₂_model.thermal_expansion),
-                                             adapt(to, pCO₂_model.haline_contraction))
+                                             adapt(to, pCO₂_model.boron_ratio))
 
 @inline function titrate_alkalinity(H, p)
     return p.DIC * (p.k¹ * H + 2 * p.k¹ * p.k²) / (H ^ 2 + p.k¹ * H + p.k¹ * p.k²) -
            (p.Alk - p.kʷ / H - p.boron / (1 + H / p.kᵇ))
 end
 
+"""
+    seawater_density(T, S)
+
+Returns the density of seawater at `T` °C and `S` psu in kg/m³ as
+per Chapter 5, Section 4.2 of Dickson, Sabine and Christian (2007, 
+http://cdiac.ornl.gov/oceans/Handbook_2007.html).
+
+With many thanks to Oscar Brandson for his implementation in cbsyst
+(https://github.com/oscarbranson/cbsyst).
+"""
+@inline function seawater_density(T, S)
```

Comment:
Should the seawater density also be a function of pressure?  What does cbsyst do for pressure?

Reply:
Maybe I should call it `surface_seawater_density`? `cbsyst` assumes that the units are per mass already so doesn't actually use the density calculation anywhere, but it also has no pressure factor.

Reply:
Would it work in the future to use multiple dispatch to have seawater_density(T, S, P) that calculates the density with pressure?  In that case we can leave it how it is now.  Otherwise surface_seawater_density is probabliy better

Reply:
why not use TEOS-10?

Reply:
That's a good idea I'll change the new PR to do that

Reply:
Looking at the oceananigans code, is there a problem with the TEOS10 that means it's better to use the polynomial approximation?

Reply:
I mean, why not use `SeawaterPolynomials`, which provides a polynomial approxmation to TEOS10?

There's no issue with TEOS10, it's just expensive so we use polynomial approximations.

Even if you don't like the polynomial approximations currently in SeawaterPolynomials, I feel it's best to contribute any new polynomial approximations to that package (so they can be used everywhere) rather than here. See the package:

https://github.com/CliMA/SeawaterPolynomials.jl

Reply:
Sorry should have updated here, I tried using SeawaterPolynomials as well as the TEOS peoples c-wrapper but then realised that it also isn't possibly to use it on GPU, so went with SeawaterPolynomials being the default.

I'm making this and a bunch of other changes in #186.

Reply:
Right, that is the purpose of SeawaterPolynomials